go.mod

```diff
@@ -208,7 +208,7 @@ require (
 	github.com/power-devops/perfstat v0.0.0-20210106213030-5aafc221ea8c // indirect
 	github.com/prometheus/common/sigv4 v0.1.0 // indirect
 	github.com/prometheus/procfs v0.9.0 // indirect
-	github.com/redis/rueidis v1.0.2-go1.18
+	github.com/redis/rueidis v1.0.10-go1.18
```

Comment:
Actually is there a reason we don't go with the non-`go1.18` version? Would we have to bump our go mod to `1.20`?

Reply:
> Actually is there a reason we don't go with the non-`go1.18` version? Would we have to bump our go mod to `1.20`?

It would be great to use the non-`go1.18` version.  But one obstacle might be the opentelemetry. The non-`go1.18` of rueidis uses opentelemetry v1.16.0